src/HealthChecks.Rabbitmq/RabbitMQHealthCheck.cs

```diff
@@ -52,8 +52,7 @@ private IConnection EnsureConnection()
                     factory = new ConnectionFactory
                     {
                         Uri = _options.ConnectionUri,
-                        AutomaticRecoveryEnabled = true,
-                        UseBackgroundThreadsForIO = true
```

Comment:
it got deprecated and removed (it was unused), see https://github.com/rabbitmq/rabbitmq-dotnet-client/pull/846 for more details